Servers/Administration/OJS.Servers.Administration/Controllers/ContestsController.cs

```diff
@@ -240,5 +252,19 @@ private async Task AddIpsToContest(Contest contest, string mergedIps)
                 await this.participantsData.InvalidateByContestAndIsOfficial(contest.Id, isOfficial: false);
             }
         }
+
+        private void AddEmptyDefaultCategory(List<FormControlViewModel> formControls)
+        {
+            var contestCategory = formControls
+                .FirstOrDefault(x => x.Type == typeof(ContestCategory));
+
+            if (contestCategory?.Options is not null)
+            {
+                var contestCategories = contestCategory.Options.ToList()!;
```

Comment:
The exclamation mark, in the end, is because the compilator requires it.

Reply:
Better put this into a comment, as it is part of the code.